packages/tyche_utils/tyche_utils.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Utilities for Tyche to visualize results produced from tests run by Mica"
+maintainer: ["Ernest Ng <ernest@cs.cornell.edu>"]
+authors: ["Ernest Ng <ernest@cs.cornell.edu>"]
+license: "ISC"
+homepage: "https://github.com/ngernest/mica"
+bug-reports: "https://github.com/ngernest/mica/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "yojson"
+  "base" {>= "v0.15.1"}
+  "core_unix" {>= "v0.15.0"}
+  "odoc" {with-doc}
```

Comment:
We have [tests (failed: Library "alcotest" not found.)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/51b56fad1033fa4ad04b706eef5074af6f223397/variant/compilers,4.11,tyche_utils.0.1.0,tests)

```
#=== ERROR while compiling tyche_utils.0.1.0 ==================================#
# context              2.3.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/ngernest/mica/archive/refs/tags/v0.1.0.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/tyche_utils.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p tyche_utils -j 39 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/tyche_utils-7-ad7cb0.env
# output-file          ~/.opam/log/tyche_utils-7-ad7cb0.out
### output ###
# File "test/utils_test/dune", line 3, characters 21-29:
# 3 |  (libraries ppx_mica alcotest ppxlib)
#                          ^^^^^^^^
# Error: Library "alcotest" not found.
# -> required by _build/default/test/utils_test/test_runner.exe
# -> required by alias test/utils_test/runtest in test/utils_test/dune:2
```

Which is just indicating that the package doesn't declare the needed test dependency on alcotest:

```suggestion
  "odoc" {with-doc}  
  "alcotest" {with-test}
```

Reply:
Thanks!